doc/appendix-de-kassensichv/appendix-de-kassensichv.md

```diff
@@ -109,30 +109,32 @@ The ftReceiptCase indicates the receipt type and defines how it should be proces
-| `0x444500000000000C` | b2b-invoice<br /> TBD: <br /> DSFinV-K: BON_TYP=Beleg, can be overwritten by ftReceiptCaseFlag  | Beleg | 1.3- |
-| `0x444500000000000D` | b2c-invoice<br /> TBD: <br /> DSFinV-K: BON_TYP=Beleg, can be overwritten by ftReceiptCaseFlag  | Beleg | 1.3- |
-| `0x444500000000000E` | info-invoice<br /> TBD: <br /> DSFinV-K: BON_TYP=AVRechnung | AVRechnung | 1.3- |
-| `0x444500000000000F` | info-delivery-note<br /> TBD: <br /> DSFinV-K: BON_TYP=AVTransfer | AVTransfer | 1.3- |
-| `0x4445000000000010` | info-order<br /> TBD: <br /> DSFinV-K: BON_TYP=AVBestellung | AVBestellung | 1.3- |
-| `0x4445000000000011` | cash deposit / cash pay-in / cash pay-out / exchange<br /> TBD: <br /> DSFinV-K: BON_TYP=Beleg, can be overwritten by ftReceiptCaseFlag | Beleg | 1.3- |
-| `0x4445000000000012` | material consumption<br /> TBD: <br /> DSFinV-K: BON_TYP=AVSachbezug | AVSachbezung | 1.3- |
-| `0x4445000000000013` | info-internal<br /> TBD: <br /> DSFinV-K: BON_TYP=AVSonstige | AVSonstige | 1.3- |
-| `0x4445000000000014` | protocol<br /> TBD: <br /> DSFinV-K: BON_TYP=AVSonstige | AVSonstige  | 1.3- |
-| `0x4445000000000015` | foreign sales<br /> TBD: <br /> DSFinV-K: BON_TYP=AVSonstige | AVSonstige | 1.3- |
+| `0x4445000000000000` | unknown type for country-code "DE"<br />This is handled like a pos-receipt (`0x4445000000000001`). | Beleg <br> Kassenbeleg-V1 | 1.3- |
+| `0x4445000000000001` | pos-receipt<br />main kind of receipt processed on a pos-system. creates a turnover and/or change in the amount of cash in the till or similar operations. <br />using the ChargeItems and PayItems to hand over details for processing. independent from the used flow (explicit/implicit), the ChargeItems and PayItems should contain the full final state of the receipt. the duration of the action is calculated using the minimum and maximum of datetime over ReceiptRequest/ChargeItems/PayItems.<br /> DSFinV-K: BON_TYP=Beleg, can be overwritten by ftReceiptCaseFlag  | Beleg <br> Kassenbeleg-V1 | 1.3- |
+| `0x4445000000000002` | zero-receipt<br />used for communication and functional test of the fiskaltrust.SecurityMechanism. In addition a detailed status information is responded on the used TSE-Device. Is only valid when charge items block (ftChargeItems) and pay items block (ftPayItems) in the ReceiptRequest are empty arrays.<br />Also TSE data are unloaded, what may cause a long running request.<br />this works only on implicit flow. calling without the ReceiptCaseFlag 0x0000000100000000 ends up in an exception.<br />if you want to end a ongoing transaction without turnover (e.g. all items on a receipt are voided) then use a regular ReceiptCase.<br /> Informations returned:<br />- List of cbReceiptReference <-> Transaction-ID<br />- Statusdata of TSE, serialnumber, available/free memory, available number of signatures left, ...<br /> DSFinV-K: BON_TYP=AVSonstige, ChargeItems and PayItems have to be empty | AVSonstige ?? [none] <br> SonstigerVorgang  | 1.3- |
```

Comment:
i would tend to not use `AVSonstige` here. would also mean doesn't end up in `Bonkopf.csv` because it doesn't contribute to turnover by definition.
same for `0x4445000000000003`, `0x4445000000000004`, `0x4445000000000005`, `0x4445000000000006`, `0x4445000000000007`

Reply:
If we use `SonstigerVorgang` we do not have to/ can not define a `BON_TYP` so I totally agree that we should not add AVSonstige for those types especially if they are not relevant for the turnover

Reply:
removed.